base/applications/rapps/include/rapps.h

```diff
@@ -14,8 +14,11 @@
 #include "configparser.h"
 
 extern LONG g_Busy;
+extern bool g_PendingInstalledViewRefresh;
```

Comment:
It looks like this global flag could be made a static public member of the CMainWindow class (or a suitable existing one).